Does not build with googletest 1.13

Both on macOS with latest Xcode 
```
[cmr-5887831f71679dcf4801a0079733d666ac06404f] [100%] Linking CXX executable cmr_gtest
[cmr-5887831f71679dcf4801a0079733d666ac06404f] Undefined symbols for architecture x86_64:
[cmr-5887831f71679dcf4801a0079733d666ac06404f]   "testing::InitGoogleTest(int*, char**)", referenced from:
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       _main in test_main.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]   "testing::AssertionResult::AssertionResult(testing::AssertionResult const&)", referenced from:
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       testing::AssertionResult testing::internal::CmpHelperOpFailure<int, unsigned long>(char const*, char const*, int const&, unsigned long const&, char const*) in test_graph.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]   "testing::AssertionFailure()", referenced from:
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       testing::AssertionResult testing::internal::CmpHelperOpFailure<int, unsigned long>(char const*, char const*, int const&, unsigned long const&, char const*) in test_graph.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]   "testing::AssertionSuccess()", referenced from:
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       ComplementTotalUnimodularity_Examples_Test::TestBody() in test_ctu.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Graph_Modifications_Test::TestBody() in test_graph.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Regular_OneSum_Test::TestBody() in test_regular.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Regular_SeriesParallelTwoSeparation_Test::TestBody() in test_regular.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Regular_NestedMinorSearchTwoSeparation_Test::TestBody() in test_regular.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Regular_R10_Test::TestBody() in test_regular.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       Regular_R12_Test::TestBody() in test_regular.cpp.o
[cmr-5887831f71679dcf4801a0079733d666ac06404f]       ...
```
and also on `ubuntu-focal` with default GCC

Any hints?

(I'm trying to add it to Sage - https://github.com/sagemath/sage/pull/35801)

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -37,7 +37,7 @@ endif()
 
 # Configure cmr_gtest target.
 target_compile_features(cmr_gtest PRIVATE cxx_auto_type)
-target_link_libraries(cmr_gtest gtest_main CMR::cmr)
+target_link_libraries(cmr_gtest gtest_main gtest CMR::cmr)
    
 include(GoogleTest)
 gtest_discover_tests(cmr_gtest)

